doc/NextGen_ON_CONUS.md

```diff
@@ -0,0 +1,174 @@
+# NextGen on CONUS
+
+This documentation provides instructions on all neccessary steps and components to run NextGen jobs at CONUS scale. Considering the computation's large scale, we focus only on running parallel jobs using MPI.
+
+* [Summary](#summary)
+* [Download the Codes](#doenload-the-codes)
```

Comment:
```suggestion
* [Download the Codes](#download-the-codes)
```

Oops, I almost missed this actually

Reply:
This one displays ok but click won't take you to the right place.